src/components/WelcomeIntro.js

```diff
@@ -1,13 +1,60 @@
 import React from "react";
-import { Col, Container, Row } from "react-bootstrap";
+import { Col, Container, Row, Button } from "react-bootstrap";
+import CocoaBean from "../assets/CocoaBean.svg";
+import JellyBean from "../assets/JellyBean.svg";
+import GardenBean from "../assets/GardenBean.svg";
+import SoyBean from "../assets/SoyBean.svg";
+import MagicBean from "../assets/MagicBean.svg";
+import CabinCupLogo from "../assets/CabinCupLogo.svg";
+
+
+import '../styles/cabincup.scss';
+
 
 const WelcomeIntro = () => {
   return (
-    <Container className="mt-5">
+    <Container>
       <Row>
-        <Col>
-          <div className="cabin">Cabin Intro</div>
+        <Col md={6}>
+          <div className="cabin-cup">
+            <Row>
+              <Col md={{span: 7, offset: 1}}>
```

Comment:
what does this syntax mean? i've just never seen it before :o

Reply:
It sets  the bootstrap column to take up 7/12ths of the row and start 1 column from the left 😄 

It's the same as doing `<div className="col-md-7 offset-md1">` (in case you've seen that syntax before).